vendor/fof/drafts/resources/locale/en.yml

```diff
@@ -21,9 +21,11 @@ fof-drafts:
       empty_text: You haven't saved any drafts
       title: Drafts
       delete_button: Delete Draft
+      deleteAll_button: Delete All Drafts
```

Comment:
delete_all_button for better consistentcy

Reply:
> delete_all_button for better consistentcy
em sửa rồi a!

Reply:
![1](https://user-images.githubusercontent.com/51877144/103225515-a0e34580-495c-11eb-990e-92d48183da64.gif)